merino-adm/src/remote_settings/mod.rs

```diff
@@ -202,6 +207,41 @@ impl RemoteSettingsSuggester {
 
         Ok(())
     }
+
+    /// Gets the "full keyword" (the suggested completion) for a query. The data
+    /// from adM doesn't include this data directly, so we make our own based on
+    /// the available keywords.
+    ///
+    /// 1. Find the first keyword phrase that has more words than the query. Use
+    ///    its first `queryWords.length` words as the full keyword. e.g., if the
```

Comment:
Could you update the doc to reflect the variables used in the function? `queryWords` and `result.keywords` are a little confusing here.